doc/cephfs/multimds.rst

```diff
@@ -225,3 +225,48 @@ For the reverse situation:
 
 The ``home/patrick`` directory and its children will be pinned to rank 2
 because its export pin overrides the policy on ``home``.
+
+
+Dynamic subtree partitioning with Balancer on specific ranks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CephFS file system provides the ``bal_rank_mask`` option to enforce the balancer
+to dynamically rebalance subtrees within particular active MDS ranks. This
+allows administrators to exploit both the dynamic subtree partitioning and
+static pining schemes in different active MDS ranks so that metadata loads
+are optimized based on users demand. For instance, in realistic cloud
+storage environments, where a lot of sub-volumes are allotted to multiple
+computing nodes (e.g., VMs and containers), some subvolumes that require
+high performance are managed by static partitioning, whereas most subvolumes
+that have moderate workload are managed by the balancer. As the balancer
+evenly spreads metadata workloads to all active MDS ranks, performances of
+static pinned subvolumes inevitably may be affected or degraded. If this option
+is configured, subtrees managed by the balancer are not affected by
+static pinned subtrees. The opposite is also true.
```

Comment:
I'd like to see more clarity on what "the opposite is also true" means, it might be interpreted in more than one way.

Reply:
I wanted to say that static pinned subtrees and dynamic partitioned subtrees managed by the balancer do not affect each other. How about eliminating "The opposite is also true."?